VideoAPI/Video.API/Controllers/InstantMessageController.cs

```diff
@@ -84,10 +84,10 @@ public async Task<IActionResult> AddInstantMessageToConferenceAsync(Guid confere
 
                 return Ok("InstantMessage saved");
             }
-            catch (ConferenceNotFoundException)
+            catch (Exception e)
             {
-                _logger.LogError($"Unable to find conference {conferenceId}");
-                return NotFound();
+                _logger.LogError(e, $"Unable to add instant messages to conference {conferenceId}");
+                return BadRequest();
```

Comment:
Maybe exception catch should not return BadRequest
Return bad request if the request object is not validated properly

Reply:
In this case, it something wrong with EF and IM to add, possibly request is bad